### ISSUE TITLE
LWIP connection limits are very low

### ISSUE THREAD

**Issue by scareything** [openziti/ziti-tunnel-sdk-c#189]
The compile-time LWIP tunables in CMakeLists.txt were set very low initially:

```
# lwip macro defaults. override on command line or in parent cmakelists.
set(LWIP_MEM_SIZE 524288 CACHE STRING "LWIP MEM_SIZE option")
set(LWIP_PBUF_POOL_SIZE 512 CACHE STRING "LWIP PBUF_POOL_SIZE option")
set(UDP_MAX_CONNECTIONS 64 CACHE STRING "LWIP MEMP_NUM_UDP_PCB option")
set(TCP_MAX_QUEUED_SEGMENTS 1024 CACHE STRING "LWIP MEMP_NUM_TCP_SEG option")
set(TCP_MAX_CONNECTIONS 256 CACHE STRING "LWIP MEMP_NUM_TCP_PCB option")
```

We should increase them to handle reasonable workloads, but doing so may require increasing other limits (e.g. `MEM_SIZE`) in lwipopts.h.

**Comment by NicFragale** [comment]
Upvote to address: set(UDP_MAX_CONNECTIONS 64 CACHE STRING "LWIP MEMP_NUM_UDP_PCB option")
It is possible this is affecting DNS resolution at (at least) ARB Midstream.

**Comment by scareything** [comment]
Closed with #230
